tests/cross_functional/kcs/test_disable_mcg_external_service.py

```diff
@@ -26,13 +26,27 @@ def patch_noobaa_object(self, request):
             resource_name="noobaa",
         )
 
-        # patch noobaa object
-        noobaa_ocp_obj.patch(
-            resource_name="noobaa",
-            params='{"spec": {"disableLoadBalancerService": true }}',
-            format_type="merge",
+        # get storagecluster object
+        storagecluster_obj = OCP(
+            kind="storagecluster",
+            namespace=config.ENV_DATA["cluster_namespace"],
+            resource_name="ocs-storagecluster",
         )
 
+        if config.ENV_DATA["deployment_type"].lower() == "managed":
```

Comment:
Given Dev's recommendation to patch the NooBaa CR rather than the StorageCluster CR, as discussed please implement this change for all deployments, not just managed ones